src/legacy/core_plugins/kibana/public/visualize/np_ready/application.ts

```diff
@@ -174,3 +178,15 @@ function createLocalI18nModule() {
     .filter('i18n', i18nFilter)
     .directive('i18nId', i18nDirective);
 }
+
+function createLocalLinkedSearchModule() {
+  angular
+    .module('app/visualize/LinkedSearch', ['react'])
+    .directive('linkedSearch', reactDirective => {
+      return reactDirective(wrapInI18nContext(LinkedSearch), [
```

Comment:
Please do not use `wrapInI18nContext`, it's a legacy dependency we have to get rid of eventually. Instead, use `coreStart.i18n.Context` as a wrapper.

In this case you can pass down `core` from `createLocalAngularModule` into `createLocalLinkedSearchModule` and then use it there:
```tsx
      return reactDirective((props: any) => <core.i18n.Context><LinkedSearch {...props} /></core.i18n.Context>, [
```

Reply:
Thanks for it! Updated!